text/pdf_using_cjk_font.go

```diff
@@ -0,0 +1,65 @@
+/*
+ * Insert text using a CJK font.
+ *
+ * Run as: go run pdf_using_cjk_font.go
+ */
+
+package main
+
+import (
+	"fmt"
+
+	"github.com/unidoc/unipdf/v3/creator"
+	"github.com/unidoc/unipdf/v3/model"
+)
+
+func main() {
+	outputFile := "output.pdf"
+
+	err := genPdfFile(outputFile)
+
+	if err != nil {
+		fmt.Println(err.Error())
+	}
+}
+
+func genPdfFile(outputFile string) error {
+	compositeFontRegular, err := model.NewCompositePdfFontFromTTFFile("./rounded-mplus-1p-regular.ttf")
+
+	if err != nil {
+		return err
+	}
+
+	c := creator.New()
+	c.SetPageMargins(50, 50, 100, 70)
+	c.EnableFontSubsetting(compositeFontRegular)
```

Comment:
Can you add a short comment about what this does and how much it reduces the file size?  Helpful for users to see, this is an important step.